### DISCUSSION TITLE
Correct citation for MET V10.0.0?

### DISCUSSION THREAD

**Discussion by jwraby** [dtcenter/METplus#1777] [Existing Builds] [answered]
I looked at the online documentation which provided the following citation for MET V10.1.2:
Newman, K., J. Opatz, T. Jensen, J. Prestopnik, H. Soh, L. Goodrich, B. Brown, R. Bullock, J. Halley Gotway, 2022: The MET Version 10.1.2 User’s Guide. Developmental Testbed Center. Available at: https://github.com/dtcenter/MET/releases

Is this correct for V10.0.0? Thanks.

**Comment by georgemccabe** [comment, marked as answer]
Hi @jwraby,

We rotate the author list for each major release. Here is the correct citation for MET v10.0.1:

> Halley Gotway, J., K. Newman, H. Soh, J. Opatz, T. Jensen, J. Prestopnik, L. Goodrich, D. Fillmore, B. Brown, R. Bullock, T. Fowler, 2021: The MET Version 10.0.1 User’s Guide. Developmental Testbed Center. Available at: https://github.com/dtcenter/MET/releases

The citation for v10.0.0 is the same except for the version number because we do not rotate the list for bugfix releases:

> Halley Gotway, J., K. Newman, H. Soh, J. Opatz, T. Jensen, J. Prestopnik, L. Goodrich, D. Fillmore, B. Brown, R. Bullock, T. Fowler, 2021: The MET Version 10.0.0 User’s Guide. Developmental Testbed Center. Available at: https://github.com/dtcenter/MET/releases

**Reply by jwraby** [reply]
Thanks, George.
